src/parser/jobs/rdm/modules/ManaStackGauge.tsx

```diff
@@ -0,0 +1,214 @@
+		[this.data.actions.ENCHANTED_ZWERCHHAU.id, {manaStack: 1}],
+		[this.data.actions.ENCHANTED_REDOUBLEMENT.id, {manaStack: 1}],
+	])
+	public spenderModifiers = new Map<number, ManaStackGaugeModifier>([
+		[this.data.actions.VERHOLY.id, {manaStack: -3}],
+		[this.data.actions.VERFLARE.id, {manaStack: -3}],
+		[this.data.actions.SCORCH.id, {manaStack: 0}],
+		[this.data.actions.RESOLUTION.id, {manaStack: 0}],
+	])
+	public gaugeBreakers: number[] = [
+		this.data.actions.VERAERO.id,
+		this.data.actions.VERAERO_II.id,
+		this.data.actions.VERAERO_III.id,
+		this.data.actions.VERSTONE.id,
+		this.data.actions.VERTHUNDER.id,
+		this.data.actions.VERTHUNDER_II.id,
+		this.data.actions.VERTHUNDER_III.id,
+		this.data.actions.VERFIRE.id,
+		this.data.actions.JOLT.id,
+		this.data.actions.JOLT_II.id,
+		this.data.actions.IMPACT.id,
+	]
+	gaugeBreak: GaugeBreaker[] = []
+	//Any loss is a major severity
+	severity = {
+		1: SEVERITY.MAJOR,
+	}
+	manaStatistics = {
+		white: {
+			overcapLoss: 0,
+		},
+		black: {
+			overcapLoss: 0,
+		},
+	}
+
+	override initialise() {
+		super.initialise()
+
+		this.addEventHook(
+			filter<Event>()
+				.type('action')
+				.source(this.parser.actor.id)
+				.action(oneOf(Array.from(this.gaugeModifiers.keys()))),
+			this.onGaugeModifying
+		)
+		this.addEventHook(
+			filter<Event>()
+				.type('action')
+				.source(this.parser.actor.id)
+				.action(oneOf(this.gaugeBreakers)),
+			this.onGaugeModifying
+		)
+		this.addEventHook(
+			filter<Event>()
+				.type('action')
+				.source(this.parser.actor.id)
+				.action(oneOf(Array.from(this.spenderModifiers.keys()))),
+			this.onGaugeSpender
+		)
+		this.addEventHook('complete', this.onComplete)
+	}
+
+	private onGaugeModifying(event: Events['action']) {
+		const modifier = this.gaugeModifiers.get(event.action)
+
+		//Always check it we should break
+		if (this.shouldGaugeBreak(event)) {
+			this.onGaugeBreak(event)
+			//No further modifications to the gauge if this skill broke it.
+			return
+		}
+
+		if (modifier == null) {
+			return
+		}
+
+		if (this.manaStackGauge.capped) {
+			const manaSpender = this.gauge.spenderModifiers.get(event.action)
+			this.debug(`Gauge is Broken by action ${this.data.getAction(event.action)?.name}`)
+			this.debug(`manaSpender is ${manaSpender} white mana value of ${manaSpender?.white} and black mana value of ${manaSpender?.black}`)
+			this.debug(`Time the Overcap happened ${this.parser.formatEpochTimestamp(event.timestamp)}`)
+			if (manaSpender != null) {
+				this.manaStatistics.white.overcapLoss += Math.abs(manaSpender.white)
+				this.manaStatistics.black.overcapLoss += Math.abs(manaSpender.black)
+			}
+		}
+
+		this.manaStackGauge.modify(modifier.manaStack)
+	}
+
+	private onGaugeSpender(event: Events['action']) {
+		const modifier = this.spenderModifiers.get(event.action)
+
+		if (modifier == null) {
+			if (this.shouldGaugeBreak(event)) {
+				this.onGaugeBreak(event)
+			}
```

Comment:
This condition can never return true, since breakers only call into the onGaugeModifier function.

Reply:
Indeed, removed the check entirely.